hack/gke/xfs-format/xfs-format.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+ephemeral_mountpoint=/mnt/stateful_partition/kube-ephemeral-ssd
+raid_device=$( awk '$2=="/mnt/stateful_partition/kube-ephemeral-ssd" {print $1}' /proc/mounts )
+raid_dir=/mnt/raid-disks/disk0
+
+if [[ -z ${raid_device} ]]; then
+  echo "Raid device not mounted"
+  exit 1
+fi
+
+if [[ "$( blkid -o value -s TYPE ${raid_device} )" == "xfs" ]]; then
+  echo "Raid device's file system is already xfs."
+  exit 0
+fi
+
+systemctl stop kube-node-configuration.service
+systemctl stop kubelet.service
+/home/kubernetes/bin/crictl stopp $( /home/kubernetes/bin/crictl pods -q )
+systemctl stop containerd.service
+
+tmp_dir=$( mktemp -d )
+cp -af ${ephemeral_mountpoint}/. ${tmp_dir}/
+
+umount $( grep -e '/home/kubernetes/containerized_mounter' -e '/var/lib/containerd' -e '/home/containerd' /proc/mounts | sed -E -e 's/([^ ]+) ([^ ]+) .+/\2/' | sort -u -r ) || true
+# Separate line due to existing dependencies
+umount $( grep -e "${raid_device}" /proc/mounts | sed -E -e 's/([^ ]+) ([^ ]+) .+/\2/' | sort -r ) || true
+mkfs.xfs -f ${raid_device}
+
+cp -af ${tmp_dir}/. ${ephemeral_mountpoint}/
+rm -rf ${tmp_dir}
+
+mkdir -p ${ephemeral_mountpoint}/scylla
```

Comment:
let's try to avoid `-p` if you substitute wrong path this can make a mess and still succeed, safer to create the particular dirs individually